doc/source/tutorials-website/create_tutorials_html.py

```diff
@@ -0,0 +1,134 @@
+import os
```

Comment:
Please also add at the beginning of this file
```
Adapted from https://github.com/qutip/qutip-tutorials/blob/1d1da2fc623372fa11ad5370a4fcd19452aad8fa/website/create_index.py
```

Reply:
> Please also add at the beginning of this file
> 
> ```
> Adapted from https://github.com/qutip/qutip-tutorials/blob/1d1da2fc623372fa11ad5370a4fcd19452aad8fa/website/create_index.py
> ```

this link or the  link to the main branch? https://github.com/qutip/qutip-tutorials/blob/main/website/create_index.py

Reply:
This link is better because it is permanent and bound to a particular commit. The main branch may change.